### DISCUSSION TITLE
Separate PDF

### DISCUSSION THREAD

**Discussion by Torle2002** [Stirling-Tools/Stirling-PDF#1924] [Q&A]
![Bildschirmfoto vom 2024-09-17 01-20-53](https://github.com/user-attachments/assets/552f2d3c-602c-4451-be58-ddfcf0e180df)
Hello everyone, I want to separate pages from the PDF over here so that it is directly divided into several documents, but if I specify it like that, then pages 1-19 are always one document with one page, how can I separate that correctly?

germen 
Hallo zusammen, ich möchte Seiten aus dem PDF trennen, so dass es direkt in mehreren Dokumenten ist, aber das macht, wenn ich dies dann Seite 1-19 alle immer ein Dokument mit einer Seite, wie kann ich es richtig trennen?

sorry ich habe einen übersetzter benutzt weil ich kein englich kann

**Comment by Frooodle** [comment]
Split splits the pdf at that page number on a 10 page doc writing 5 splits it at the number 5 so you have 2 docs each 5 pages doing 3-5 generates the numbers 3,4,5 doing the same logic it splits the pdf so you have page 
1,2
3
4
5
6789
Etc

**Comment by Frooodle** [comment]
TLDR 1,3,5,9,11 etc should work for you?

**Reply by Torle2002** [reply]
Yes, it worked, thank you
